security.md

```diff
@@ -4,35 +4,39 @@ title: Reporting a Security Vulnerability
 permalink: /security/
 ---
 
-# Reporting a Security Vulnerability
+# Security Policy
```

Comment:
Is this a duplicate of the work in https://github.com/ballerina-platform/ballerina-dev-website/pull/180/ ?

Reply:
Closed #180.